.github/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 <h3>New features since last release</h3>
 
+* TBD: Added custom gate application for Toffoli in `default.qubit` [(#1249)](https://github.com/PennyLaneAI/pennylane/pull/1249) [(#1215)](https://github.com/PennyLaneAI/pennylane/issues/1215)
```

Comment:
Just a couple of minor things here :slightly_smiling_face: Maybe it's also more fitting for the `Improvements` section, could you add it there?
```suggestion
* Added custom gate application for Toffoli in `default.qubit`.
  [(#1249)](https://github.com/PennyLaneAI/pennylane/pull/1249)
```